### DISCUSSION TITLE
How to change formats date in Datepicker popup

### DISCUSSION THREAD

**Discussion by ryunakai** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2681] [Q&A] [answered]
Hello,

Is it possible to change date formats displayed in DatePicker popup?

I am using this tool kit in Japanese language. I understand how to change the language seeing a demo,
but a date format in this toolkit is not common in our language.
We use a style like yyyy/MM/dd (ddd), the day of the week is written after the date.

I would like to know if there is a way to change it.

![datepicker](https://user-images.githubusercontent.com/6489782/51128638-b47a6c00-186b-11e9-89bb-4cc267170e4d.png)

**Comment by donvan** [comment]
hi, how to change Datepicker language

**Comment by ryunakai** [comment, marked as answer]
@donvan 
hi,
set a Language attribute in a xaml file.
for example Japanese locale.

`
<DatePicker 
                Language="ja-JP"
                />
`
